selfservice/flow/settings/handler_test.go

```diff
@@ -60,12 +60,18 @@ func TestHandler(t *testing.T) {
 	_ = testhelpers.NewErrorTestServer(t, reg)
 	_ = testhelpers.NewSettingsUIFlowEchoServer(t, reg)
 
-	conf.MustSet(ctx, config.ViperKeySelfServiceSettingsPrivilegedAuthenticationAfter, "1ns")
```

Comment:
This set all sessions in this test suite to be unprivileged by default. This similar behavior is achieved by creating the `primaryUser` and `otherUser` clients using the `testhelpers.UnprivilegedProvider()`